### ISSUE TITLE
Detect acupunct point text like Lu1

### ISSUE THREAD

**Issue by christophpickl** [christophpickl/gadsu#81]
* when enter text, parse if its a accupuncture point, eg "Gb25" or meridian "Gb"
* make it a clickable link and open accupuncture database window => depends on #12 acupuncture database

~~could use tags (as already implemented for rich text editor) to store parsed tokens~~

OUT OF SCOPE

* recognize internal abbriviations like "BL", "RL", "SP"
* hover tooltip popup with short description

**Comment by christophpickl** [comment]
done
